sdk/storage/azure-storage-file-datalake/tests/test_directory.py

```diff
@@ -1054,27 +1293,33 @@ def test_rename_to_an_non_existing_directory_in_another_file_system(self, datala
 
 
         # the source directory has been renamed to destination directory, so it cannot be found
-        with self.assertRaises(HttpResponseError):
+        with pytest.raises(HttpResponseError):
             source_directory_client.get_directory_properties()
 
-        self.assertEqual(non_existing_dir_name, res.path_name)
+        assert non_existing_dir_name == res.path_name
 
     @pytest.mark.skip(reason="Investigate why renaming non-empty directory doesn't work")
```

Comment:
Did you try this test out? Async as well.

Reply:
Yup, still broken 😢 it is documented in our big GitHub issue that is tracking all of these skipped tests, so it will be fixed... one day